### DISCUSSION TITLE
Panaroma maps with Meaningful filenames

### DISCUSSION THREAD

**Discussion by gabortakacs78** [sni/thruk#1081] [Ideas]
Hi Sven,

I guess it would be great if we can use Panorma maps with meaningful filenames also.
Now Thruk displays only Panorama files where filename is [0-9].tab

I suggest to modify it and it should possible tu use any stadard chars in panorama map filenames.

Br,

Taki

**Comment by sni** [comment]
i put this feature is on the todo list.

**Comment by sni** [comment]
did not forget this one... Next release will support arbitrary filenames: 928ab695f22f6251784450d783b89fe1a8a46646
